### ISSUE TITLE
Scroll bars are hard to see

### ISSUE THREAD

**Issue by akihikodaki** [tootsuite/mastodon#4518]
On WebKit base browsers, the color of scroll bars is #313543 (bluish black). The color of background is #282c37 (black).
They are hard to distinguish and pixiv, the owner of Pawoo instance, received some reports from users that it is problematic for them. I also worry that that could be harder to see for people who are color blind.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Cassolotl** [comment]
We now have a high contrast theme available, and the scrollbars have been made wider in all themes too. Here's the scrollbars in the high contrast theme:

![screen shot 2018-05-26 at 12 59 05](https://user-images.githubusercontent.com/6607512/40575855-99e67440-60e4-11e8-9dc8-395f19cf0ec3.png)

Is this better?

**Comment by akihikodaki** [comment]
I am not color-blind, so I cannot really answer that, but the wider scroll bar certainly looks good.
A high contrast theme, on the other hand, does not change the color of scroll bar. I'm not sure what the theme was created for, but it changes the color of obvious buttons and is not really for accessibility in my opinion. Maybe we can close this issue when colors of such UI parts changed.
